src/layouts/sidebar-sidecar/sidebar-sidecar-layout.module.css

```diff
@@ -68,6 +68,12 @@ for further details.
   max-width: calc(780px + (var(--main-area-padding-left-right) * 2));
   padding-left: var(--main-area-padding-left-right);
   padding-right: var(--main-area-padding-left-right);
+
+  @media (--dev-dot-tablet-down) {
+    padding-left: 0;
+    padding-right: 0;
+    max-width: none;
+  }
```

Comment:
This reduces the gutter on mobile from 48px to 24px. This felt like the best place to do that, but open to somewhere else!

Reply:
This is the right place! 🙌🏻